### ISSUE TITLE
pkgconf dependency version

### ISSUE THREAD

**Issue by masterspline** [build2/build2#42]
build2 in manifest declares dependancy
`depends: libpkgconf [1.4.2 1.6.0-)`
but [debian buster use pkgconf version 1.6.0](https://packages.debian.org/buster/pkgconf). [Fedora package](https://src.fedoraproject.org/rpms/build2/blob/master/f/build2.spec) by @mkrupcale also use system [package version 1.6.1](https://apps.fedoraproject.org/packages/pkgconf)
```
BuildRequires:  libpkgconf-devel
...
BuildRequires:  pkgconf
```
Is dependency for libpkgconf 1.6.0-) correct?

**Comment by boris-kolpackov** [comment]
`libpkgconf` maintainers were playing a bit loose with backwards-compatibility guarantees (see https://git.dereferenced.org/pkgconf/pkgconf/issues/15). Plus there were some other implementation-related breakages. So we've decided to be conservative and only allow versions that we have reviewed/tested ourselves.

We will try upgrading to 1.6.0 in the next couple of days and if everything looks good, will adjust the version constraint for the next release.

**Comment by boris-kolpackov** [comment]
We've tried the `libpkgconf` 1.6.1 package: there was one memory leak that was fixed in the just-released 1.6.2. So the next release of `build2` will have the adjusted version constraint in the package `manifest` (this change is already on stage).

I am closing this issue but feel free to reopen it if there are still any problems.
